python/requirements_compiled.txt

```diff
@@ -1519,7 +1516,7 @@ py-cpuinfo==9.0.0
     # via deepspeed
 py-partiql-parser==0.5.0
     # via moto
-py-spy==0.4.0 ; python_version < "3.12"
+py-spy==0.3.14
```

Comment:
@hongpeng-guo I think we need to rebase/pull in the latest compiled dependencies here. This was modified in https://github.com/ray-project/ray/commit/73a4c42044e27637d1661045477373e1472858f6

Reply:
Nice catch!